verify_email.ml

```diff
@@ -0,0 +1,103 @@
+                             [
+                               a_class
+                                 [
+                                   "rounded-full bg-primary-200 w-32 h-32 flex \
+                                    justify-center items-center";
+                                 ];
+                             ]
+                           [
+                             i
+                               ~a:
+                                 [
+                                   a_class
+                                     [
+                                       "fa-solid fa-envelope text-7xl \
+                                        text-primary-500";
+                                     ];
+                                 ]
+                               [];
+                           ];
+                       ];
+                     div
+                       ~a:[ a_class [ "my-5" ] ]
+                       [
+                         h1
+                           ~a:[ a_class [ "text-3xl font-bold" ] ]
+                           [ txt "Please verify your email" ];
+                         p [ txt "You're almost there! We sent an email to" ];
+                         p
+                           ~a:[ a_class [ "font-bold text-xl" ] ]
+                           [ txt user.email ];
+                       ];
+                     div
+                       ~a:[ a_class [ "my-3" ] ]
+                       [
+                         p
+                           [
+                             txt
+                               "Just click on the link in that email to \
+                                complete your signup.";
+                           ];
+                         p
+                           [
+                             txt "If you don't see it, you may need to";
+                             span
+                               ~a:[ a_class [ "font-bold" ] ]
+                               [ txt " check your spam " ];
+                             txt "folder.";
+                           ];
+                       ];
+                     div
+                       ~a:[ a_class [ "my-3" ] ]
+                       [ p [ txt "Still can't find the email? No problem." ] ];
+                     div
+                       ~a:[ a_class [ "my-10" ] ]
+                       [
+                         a
+                           ~a:
+                             [
+                               a_class
+                                 [
+                                   "py-3 px-2 bg-primary-500 rounded \
+                                    text-gray-100 font-semibold cursor-pointer";
+                                 ];
+                             ]
+                           [ txt "Resend Verification Email" ];
+                       ];
+                   ];
+               ];
+             Footer_layout.footer;
+             Tyxml.Html.Unsafe.data "";
```

Comment:
This will be corrected with a function to call our backend when the user wants to resend a verification email, in case the one they received already expired. :)